apps/trip_plan/lib/trip_plan/transit_detail.ex

```diff
@@ -16,6 +16,7 @@ defmodule TripPlan.TransitDetail do
 
   @type fares :: %{
           highest_one_way_fare: Fare.t(),
-          lowest_one_way_fare: Fare.t()
+          lowest_one_way_fare: Fare.t(),
+          reduced_one_way_fares: [Fare.t()]
```

Comment:
Are there any cases for which there are multiple reduced fares for a given route or trip? If I'm interpreting the designs correctly it seems like they expect to only display one or none.

If that's the case it might be easier for the front-end if we provide a single value here instead of a list. If there is never more than one result anyway, maybe it's as simple as adding a `List.first()` after line 65 in `highest_lowest_fare.ex` or something. If there are sometimes multiple results, maybe we need to clarify with Ingrid?

Reply:
I was also confused about that...from what I saw, type `reduced` that can have these values:
`@type reduced :: nil | :student | :senior_disabled | :any`
I saw that although the fares for students and senior citizens is the same price, it is returned as separate objects (one for each), so querying without the 'reduced' filter returns more than one value.
If it makes more sense to return just a unique value symbolizing a "one-way reduced fare" I can do that, just let me know.

Reply:
Interesting. @ingridpierre Do you have thoughts on how you want these displayed?

Reply:
The reduced fare is the same regardless of the different ways you get it. One unified reduced fare only is correct.

Reply:
Sounds good, I'll make those changes.